main_test.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestPrintObject(t *testing.T) {
```

Comment:
I included those test cases because I don't have kubectl in my environment. They were meant to help visualize output changes. However, I understand your point about their redundancy since they only cover non-failing scenarios. I will proceed with removing those tests as suggested.

Reply:
Yeah it's not ideal to print directly stdout much in these funcs either, it requires human assessment of the output. We could definitely test each of these utilities independently, and the final output independently without humans in the loop had we architected the code better.